assets/js/modules/analytics/setup.js

```diff
@@ -674,6 +682,18 @@ class AnalyticsSetup extends Component {
 						</p>
 					</div>
 				}
+				{ onSettingsPage && useSnippet && ( ! ampEnabled || ( ampEnabled && ! ampClientIDOptIn ) ) && (
+					<div className="googlesitekit-setup-module__input">
+						<Switch
+							id="anonymizeIP"
+							label={ __( 'Anonymize IP addresses', 'google-site-kit' ) }
+							onClick={ this.switchStatus( 'anonymizeIP' ) }
+							checked={ anonymizeIP }
+							hideLabel={ false }
+						/>
+						<p>{ anonymizeIP ? __( 'IP addresses will be anonymized.', 'google-site-kit' ) : __( 'IP addresses will not be anonymized.', 'google-site-kit' ) } <Link href="https://support.google.com/analytics/answer/2763052" external inherit>{ __( 'Learn more about IP anonymization.', 'google-site-kit' ) }</Link></p>
```

Comment:
See @aaemnnosttv's comment, let's simplify the link test to just "Learn more". Also, let's break this into multiple lines, it's too much in a single line like this.